app/views/AssociatedCompaniesView.scala.html

```diff
@@ -36,7 +36,7 @@
 
 @heading= @{AssociatedCompaniesViewHelper.heading(associatedCompaniesParameter)}
 
-@layout(pageTitle = title(form, messages("associatedCompanies.title")), backLink = BackLinkHelper.backLinkOrDefault(routes.DistributionController.onPageLoad(NormalMode).path(), mode)) {
+@layout(pageTitle = heading, backLink = BackLinkHelper.backLinkOrDefault(routes.DistributionController.onPageLoad(NormalMode).path(), mode)) {
```

Comment:
this is going to be missing the error prefix when in error as it's no longer calling the `title()` helper method. When using the `title()` method it is expecting a message key for the title, but because `heading` is a fully resolved message then the existing `title()` function may need modifying to allow a fully resolved message to be passed in instead of a message key e.g. new param called `titleString: Option[String]` that you could use to update the helper function with, maybe something like `titleString.getOrElse(messages(title))`

Reply:
fixed on [1c355e4](https://github.com/hmrc/marginal-relief-calculator-frontend/pull/278/commits/1c355e4959b4b1f511a172631ef9a25f61555435)